src/mixer/audio.ts

```diff
@@ -182,23 +186,27 @@ export class AudioEngine extends ((EngineEmitter as unknown) as {
   micMedia: MediaStream | null = null;
   micSource: MediaStreamAudioSourceNode | null = null;
   micCalibrationGain: GainNode;
-  micPrecompAnalyser: AnalyserNode;
+  micPrecompAnalyser: StereoAnalyserNode;
   micCompressor: DynamicsCompressorNode;
   micMixGain: GainNode;
-  micFinalAnalyser: AnalyserNode;
+  micFinalAnalyser: StereoAnalyserNode;
 
   finalCompressor: DynamicsCompressorNode;
   streamingDestination: MediaStreamAudioDestinationNode;
 
-  streamingAnalyser: AnalyserNode;
+  player0Analyser: StereoAnalyserNode;
+  player1Analyser: StereoAnalyserNode;
+  player2Analyser: StereoAnalyserNode;
+  playerAnalysers: StereoAnalyserNode[];
```

Comment:
Any reason for having both the individual variables, as well as the array? Wouldn't it be clearer to only have that? (For comparison, in MixerState the individual player states are just an array.)

Reply:
Hmm, true.

Reply:
Completed in #604c9c711c764028338323d5552aececae9653d3